README.md

```diff
@@ -1,2 +1,149 @@
+- **About Page:A linked webpage to Home Page giving more information about the event**
+- **Functionality: The mobile navbar, featured Speaker section were made easy for users to navigation the webpage with Javascript**
+- **Social Contact: A form allowing visitors to get in touch with me.**
+- **Responsive Design: The website is optimized for different devices and screen sizes.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link]()
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. A code editor or an IDE installed on your computer (e.g., Visual Studio Code, Sublime Text, Atom).
+2. Git installed on your system to clone the repository.
+3. A web browser to view and test the project.
+
+
+### Setup
+
+1. Open your preferred terminal or command prompt.
+2. Navigate to the directory where you want to clone the project.
+
+  Run this command;
+```sh
+  cd my-folder
+  git@github.com:Ayokunnumi1/Conference-Page.git
+```
+
+### Install
+
+
+After cloning the repository, navigate into the project's directory. If there are any dependencies required for the project, you need to install them. Typically, this involves using a package manager like npm or yarn.
+
+  Run this command;
+```sh
+  npm install
+```
+
+### Usage
+
+1. Open the index.html file in your preferred web browser.
+2. Explore the different sections of the website.
+3. Customize the content and design to match your own portfolio by modifying the HTML, CSS & JavaScript files.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+👤**Ayokunnumi Omololu**
+
+- GitHub: [@Ayokunnumi1][https://github.com/Ayokunnumi1]
+- Twitter: [@Ayokunnumi1] [https://twitter.com/AyokunnumiA]
+- LinkedIn: [Ayokunnumi] [https://www.linkedin.com/in/ayokunnumiomololu]
+
+ 
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+ 
+ ## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Project Showcase**
+- [ ] **Testimonials and recommendations**
+- [ ] **Interactive sections**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+
+ ## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project. show your support by following my github account.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I give thanks to the Microverse team for creating this project.
```

Comment:
- [ ] Your "Acknowledgements" section should gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).